src/screens/game/game.screen.tsx

```diff
@@ -222,8 +232,8 @@ export const GameScreen = () => {
 };
 
 const Screen = styled.div`
-  width: 100vw;
-  height: 100vh;
+  width: 100svw;
```

Comment:
Thanks for fixing this issue. I would've gone with a nom package, didn't know those are now built in. 👍🏻